src/fieldValidators.js

```diff
@@ -2,71 +2,74 @@
 import curry from 'ramda/src/curry';
 
 /**
-  * @desc Test if a value is ok for a required field.
-  * If the field has a formattedAddress key, it's a location, so we make sure that this key is set.
-  * If field is an array, check length.
-  * @param {Any} field - The value of the field
-  * @return {bool} True if the value is falsy
-  */
+ * @desc Test if a value is ok for a required field.
+ * If the field has a formattedAddress key, it's a location, so we make sure that this key is set.
+ * If field is an array, check length.
+ * @param {Any} field - The value of the field
+ * @return {bool} True if the value is falsy
+ */
 const testMissingValue = (field: any): string | null =>
-    (!field && field !== false) || field.length === 0 || field.formattedAddress === null
+    (!field && field !== false) ||
+    field.length === 0 ||
+    field.formattedAddress === null ||
+    (typeof field === 'string' && !field.trim())
```

Comment:
```suggestion
    (typeof field === 'string' && filed.match(/^\s*$/) !== null;)
```

With this, we can even catch tabulations :)

Reply:
Normalement, .trim() retire également les tabulations

Reply:
https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/String/trim
C'est vrai ✅